site/content/faq/100-version-numbers.md

```diff
@@ -0,0 +1,7 @@
+---
+question: Sapper is not 1.0 yet. Is it production ready and should I use it?
+---
+
+The version number reflects that there are major features we'd like to add still. E.g. TypeScript support was added in Sapper 0.28.
+
+Sapper is operationally stable. For example, some of the Svelte maintainers run their businesses on Sapper.
```

Comment:
I think it's worth including something about:

 Sapper is operationally stable. "production readiness" is subjective, and if Sapper fulfils your requirements, then it's production ready. As with any software, a major version number is not an indicator of whether something is suitable for your specific use-case.

Reply:
done